pkg/api/customization/pipeline/pipeline_execution.go

```diff
@@ -41,26 +53,65 @@ func (h *ExecutionHandler) ActionHandler(actionName string, action *types.Action
 }
 
 func (h *ExecutionHandler) rerun(apiContext *types.APIContext) error {
-	return nil
-}
+	ns, name := ref.Parse(apiContext.ID)
+	execution, err := h.PipelineExecutionLister.Get(ns, name)
+	if err != nil {
```

Comment:
I think it's not neccessary to do that because for not found error it is checked before invoking the action handler: https://github.com/rancher/norman/blob/master/api/server.go#L250